js/views/chatHeadsVw.js

```diff
@@ -88,7 +103,12 @@ module.exports = baseVw.extend({
       );
     });
 
-    this.nextChatToShow = this.nextChatToShow >= totalChats ? this.nextChatToShow : this.nextChatToShow + this.showPerScroll;
+
+    this.nextChatToShow = this.nextChatToShow >= this.collection.length ? this.nextChatToShow : this.nextChatToShow + this.showPerScroll;
+  },
+
+  remove: function(){
```

Comment:
I meant find a view that extends from baseVw which also overrides remove like you do so you can see an example of how to call the baseVw's remove (it's essential a `super` call which has a bit of a weird syntax in JS).

No, the baseVw won't call the remove of it's child. It works in the different direction. When you call childObject.remove(), if that child implements remove, it's remove will be called. It is up to the child then to ensure the baseVw remove is being called so the inheritance structure is in tact. In our case, the baseVw remove implements some functionality that we don't want to blow away.